app/models/exceptions/resource_forbidden.rb

```diff
@@ -0,0 +1,5 @@
+class ResourceForbidden < ActionController::ActionControllerError
+  def status
+    403
```

Comment:
Alternatywnie można pisać do wszystkich błędów prawie identyczne handlery, różniące się tylko kodem błędu. Zmiana nazwy to nie problem.